gitops-connector - CrashLoopBackOff

The GitOps-Connector pod fails to initiate, likely stemming from an issue related to Werkzeug/Flask version compatibility or dependencies.

```
k logs gitops-connector-696d8cdd-hvvsv -n flux-system
[2024-03-18 16:15:39 +0000] [7] [INFO] Starting gunicorn 20.0.4
[2024-03-18 16:15:39 +0000] [7] [INFO] Listening at: http://0.0.0.0:8080 (7)
[2024-03-18 16:15:39 +0000] [7] [INFO] Using worker: sync
[2024-03-18 16:15:39 +0000] [8] [INFO] Booting worker with pid: 8
[2024-03-18 16:15:39 +0000] [8] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 119, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 144, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 49, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 358, in import_app
    mod = importlib.import_module(module)
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/app/gitops_event_handler.py", line 4, in <module>
    from flask import Flask, request
  File "/usr/local/lib/python3.9/site-packages/flask/__init__.py", line 7, in <module>
    from .app import Flask as Flask
  File "/usr/local/lib/python3.9/site-packages/flask/app.py", line 27, in <module>
    from . import cli
  File "/usr/local/lib/python3.9/site-packages/flask/cli.py", line 17, in <module>
    from .helpers import get_debug_flag
  File "/usr/local/lib/python3.9/site-packages/flask/helpers.py", line 14, in <module>
    from werkzeug.urls import url_quote
ImportError: cannot import name 'url_quote' from 'werkzeug.urls' (/usr/local/lib/python3.9/site-packages/werkzeug/urls.py)
[2024-03-18 16:15:39 +0000] [8] [INFO] Worker exiting (pid: 8)
[2024-03-18 16:15:39 +0000] [7] [INFO] Shutting down: Master
[2024-03-18 16:15:39 +0000] [7] [INFO] Reason: Worker failed to boot.
```

--- a/src/requirements.txt
+++ b/src/requirements.txt
@@ -1,4 +1,4 @@
-Flask==2.1.0
+Flask==2.3
 gunicorn==20.0.4
 requests
 timeloop

